package.json

```diff
@@ -25,5 +25,9 @@
     "husky": "^8.0.3",
     "prettier": "^3.0.1",
     "typescript": "^5.1.6"
+  },
+  "dependencies": {
```

Comment:
I added those as dependencies, because it doesn't match the template purposes and should be removed in that case :)